src/Provider/Wurfl.php

```diff
@@ -57,6 +57,66 @@ public function getParser()
     }
 
     /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    private function isRealBrand($value)
+    {
+        if ($value === '' || $value === null) {
+            return false;
+        }
+
+        if ($value == 'Generic') {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    private function isRealModel($value)
```

Comment:
@NielsLeenheer i like this approach. If a specific "default value" is just in one return value, we should check it their and not in the generic `isRealResult` 

I will update some parts with this approach.